library/include/ck/library/utility/check_err.hpp

```diff
@@ -66,6 +66,7 @@ check_err(const Range& out,
     if(!res)
     {
         std::cerr << std::setw(12) << std::setprecision(7) << "max err: " << max_err << std::endl;
+        std::cerr << std::setw(12) << "number of errors: " << err_count << std::endl;
```

Comment:
One suggestion: Maybe we could also print how many % of the values are incorrect? It's often useful to know whether we have exactly 50% of wrong values or just some random 42.314%.

Reply:
Good idea, added :)